lib/fluent/plugin/in_tail.rb

```diff
@@ -92,6 +88,8 @@ def configure(conf)
 
       configure_parser(conf)
       configure_tag
+      @encoding = parse_encoding_param(conf['encoding'])
+      @from_encoding = parse_encoding_param(conf['from_encoding'])
```

Comment:
`@from_encoding` looks to require that `@encoding` is configured.
It should be checked at here.